lmdeploy/pytorch/engine/model_agent.py

```diff
@@ -742,8 +688,9 @@ async def async_forward(self, inputs: ModelInputs, swap_in_map: SwapMap,
         output = self._forward_impl(inputs,
                                     swap_in_map=swap_in_map,
                                     swap_out_map=swap_out_map)
-        await asyncio.get_event_loop().run_in_executor(None,
-                                                       self.stream.synchronize)
+        await asyncio.sleep(0)
```

Comment:
How does this method outperform the previous one?

Reply:
`asyncio.sleep` release CPU without creating new thread